presto-influx/src/test/java/io/prestosql/plugin/influx/TestingInfluxServer.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.influx;
+
+import org.testcontainers.containers.InfluxDBContainer;
+
+import java.time.Instant;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+
+public class TestingInfluxServer
+{
+    public static final String DATABASE = "Test";
+    public static final String RETENTION_POLICY = "Schema";
+    public static final String MEASUREMENT = "Data";
+    private static final Instant[] T = new Instant[] {
+            Instant.parse("2019-12-10T21:00:04.446Z"),
+            Instant.parse("2019-12-10T21:00:20.446Z"),
+            Instant.parse("2019-12-10T22:00:04.446Z"),
+    };
+    private static final String[] K1 = new String[] {"a", "b", "d"};
+    private static final String[] K2 = new String[] {"b", "c", "b"};
+    private static final double[] V1 = new double[] {1, 3, 5};
+    private static final double[] V2 = new double[] {2, 4, 6};
+
+    private final InfluxDBContainer dockerContainer;
+
+    public TestingInfluxServer()
+    {
+        dockerContainer = new InfluxDBContainer()
+                .withDatabase(DATABASE)
+                .withAuthEnabled(false);
+        dockerContainer.start();
+
+        InfluxHttp http = new InfluxHttp(getHost(), getPort(), false, DATABASE, null, null);
+        //http.executeDDL("CREATE DATABASE " + DATABASE);
+        http.execute("CREATE RETENTION POLICY " + RETENTION_POLICY + " ON " + DATABASE + " DURATION INF REPLICATION 1");
+        http.write(RETENTION_POLICY, getWrite(0), getWrite(1), getWrite(2));
+    }
+
+    public String getHost()
+    {
+        return dockerContainer.getContainerIpAddress();
+    }
+
+    public int getPort()
+    {
+        return dockerContainer.getMappedPort(InfluxDBContainer.INFLUXDB_PORT);
+    }
+
+    public static String getWrite(int row)
+    {
+        return MEASUREMENT + ",k1=" + K1[row] + ",k2=" + K2[row] + " v1=" + V1[row] + ",v2=" + V2[row] + " " + T[row].toEpochMilli() + "000000";
```

Comment:
Can we use meaningful names instead of abbreviations (k1, v2) and make this `format`?

Reply:
I've changed from k to 'tag' and v to 'field', as per influx terminology.  Or did you mean to use test data that was pretending to be measurements from a sensor or pageviews or something?

Reply:
Sorry, my comment was insufficient. 

> Or did you mean to use test data that was pretending to be measurements from a sensor or pageviews or something?

Yes, can we load TPCH data as other tests (e.g. `KafkaLoader`, `ElasticsearchLoader`)?

Reply:
Influx can't really represent that kind of data, nor run those kinds of queries, nor be used for that kind of workload?

Before using the docker container, the test was actually checking the influxql generated matched expectations, and then it was useful to know which columns were tags and which were fields.  Now the test is black box, I'm thinking even that isn't important any more.  The columns might as well be called col1, col2, ...?

Reply:
As we talked offline, we can append columns to TPCH tables according to the connector as Kafka connector. If you face issues during implementing the loader, please feel to ask about it.